### PR TITLE
Citations

### PR DESCRIPTION
**Pull request by cuahsimarko**
Draft for discussion of resource metadata model

### CODE REVIEW

**Comment by hydrocheck** [comment]
| METRIC | VALUE |
|:-----:|:-----:|

[https://sonarqube.cuahsi-workstation.com:9000/dashboard?id=hydroshare-4016](https://sonarqube.cuahsi-workstation.com:9000/dashboard?id=hydroshare-4016)

File: hs_core/migrations/0051_auto_20201021_2357.py [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+import django.db.models.deletion
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('contenttypes', '0002_remove_content_type_name'),
+        ('hs_core', '0050_auto_20200611_1912'),
+    ]
+
```
Comment (cuahsimarko) [inline hs_core/migrations/0051_auto_20201021_2357.py:15]: I had conflicts with migrations. I may need to start over and redo the migration clean.

File: hs_core/migrations/0052_tasknotification_username.py [hunk 2]
```diff
@@ -8,7 +8,7 @@
 class Migration(migrations.Migration):
 
     dependencies = [
-        ('hs_core', '0051_tasknotification'),
+        ('hs_core', '0052_tasknotification_username'),
     ]
```
Comment (cuahsimarko) [inline hs_core/migrations/0052_tasknotification_username.py:12]: Do I need to just start over with 0051 - 0054 migrations?

File: pytest/conftest.py [hunk 3]
```diff
@@ -78,7 +78,7 @@ def base_sample_resource(username='admin', title=str(uuid.uuid4()), contributor=
         owner=user,
         title=title,
         metadata=metadata,
-        files=(open('pytest.ini', 'rb'),)
+        # files=(open('pytest.ini', 'rb'),)
     )
```
Comment (cuahsimarko) [inline pytest/conftest.py:82]: This varies on filesystems and is unused, anyway. To make it more portable for now, until this is generalized, it's commented out. Ideally, leave it commented as a stub for people to pick up and work on quicker in the future.

**Comment by cuahsimarko** [comment]
There was a hidden issue with missing migrations in the develop branch. Redoing this work for clarity. Closing and will open a different PR, under a different branch.

**Pull request closed without merging**